enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -0,0 +1,177 @@
+Machine Config operator (WMCO) and a Windows Machine Config Bootstrapper (WMCB)
+binary. WMCO will collect information from the cluster and transfer it to the
+Windows node. The WMCB will then perform the necessary steps on the node for it
+to join the cluster. [WMCB](https://github.com/openshift/windows-machine-config-bootstrapper)
+has already been developed as part of the Windows Containers
+[dev preview](https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/tools/ansible/docs/ocp-4-4-with-windows-server.md).
+
+### Justification
+
+The reason we are using a WMCB binary rather than a container image is
+that all Windows container images are packaged with a Windows kernel and Red Hat
+has a policy to not ship 3rd party kernels for support reasons.
+
+### Design Details
+
+The WMCO image will be packaged with the following binaries:
+* Kubelet
+* CNI plugins
+* Hybrid-overlay
+* WMCB
+* Kube-proxy
+
+WMCO will be published to OperatorHub as a Red Hat operator. The way the cluster
+admin will enable Windows workloads on their cluster would be to first install
+WMCO. WMCO expects the cluster admin to create a predetermined Secret in its
+namespace containing a private key that will be used to interact with the
+Windows instance. WMCO will check for this Secret during boot up time and create
+a user data Secret which the cluster admin can reference in the Windows
+MachineSet that they create as described below. WMCO will populate the user data
+secret with a public key that corresponds to the private key. In addition it
+will have all the pre-requisite steps needed for setting the VM up for key based
+SSH connections.
+
+To express that they want to bring up Windows worker nodes, they will
+create a MachineSet where they reference a Windows OS image which has the
+Docker container runtime add-on enabled. They will also specify the user data
+secret created by WMCO as described above. WMCO will watch for Machine objects
+with a special label that indicate that they map to Windows VMs. Once the
+Machine object is in the "provisioned" state, WMCO will connect to the
+associated VM to transfer the binaries and perform the following steps for it to
+join the cluster as a worker node:
+* Transfer the required binaries
+* Configure the kubelet service using WMCB
+* Run the hybrid-overlay-node.exe binary
+* Configure the kubelet service for CNI using WMCB
+* Configure kube-proxy
+
+### User Stories
+
+The user stories are collected using the links below:
+* (Windows Containers Productization Epic) https://issues.redhat.com/browse/WINC-182
+* (Machine API migration)[https://issues.redhat.com/issues/?filter=12347681]
+* (Filter out "worker" windows node from being tagged by the MCC)[https://issues.redhat.com/browse/GRPA-2213]
+
+### Risks and Mitigations
+
+The main risk is that OVN Hybrid networking will not GA in 4.6. In that scenario
+a 3rd part networking solution that has been validated for all the targeted
```

Comment:
`3rd part` -> `third/3rd party`

Reply:
I'm saying either third or 3rd in addition to the `part` fix. Not saying to include that exact string

Reply:
I'd recommend `third-party`, as this is the standard way we use it throughout our official docs.